src/SignalR/clients/ts/FunctionalTests/scripts/run-tests.ts

```diff
@@ -171,6 +171,9 @@ const configFile = sauce ?
     path.resolve(__dirname, "karma.local.conf.js");
 debug(`Loading Karma config file: ${configFile}`);
 
+// Workaround for 'wd' not installing correctly. https://github.com/karma-runner/karma-sauce-launcher/issues/117
+exec("node ../node_modules/wd/scripts/build-browser-scripts.js");
```

Comment:
Are we going to need this in preview 7? If yes, retarget to `release/3.0-preview7` and start the QB process.

Reply:
It depends on how flaky the builds are. I haven't seen this fail enough yet

Reply:
WFM